README.md

```diff
@@ -85,20 +85,40 @@ There's also an easier option to pull image from docker hub then simply run with
 $ docker run -d --net=host --name tangle-accelerator wusyong/tangel-accelerator:latest
 ```
 
+### Optional: Build and Push Docker Image to Docker Hub
+
+Before pushing the docker image to Docker Hub, you need to log in the docker registry:
+
+```
+$ docker login
+```
+
+Then you could push the docker image with the following command:
+
+```
+$ make && bazel run //accelerator:push_docker
+# or
+$ make && bazel run //accelerator:push_docker --host_force_python=PY2
+```
+
 ## Developing
 
 The codebase of this repository follows [Google's C++ guidelines](https://google.github.io/styleguide/cppguide.html):
+
 - Please run `hooks/autohook.sh install` after initial checkout.
 - Pass `-c dbg` for building with debug symbols.
 
 ### Tools required for running git commit hook
+
```

Comment:
Why?

Reply:
It comes from my markdown lint tool, it automates add the empty line after the title.

Reply:
Maybe we need to lint the Readme markdown style in another pull request?

Reply:
Minimize the changes first and then introduce indention tools.

Reply:
Fixed.